PHANI-1/Scrum.txt

```diff
@@ -0,0 +1,132 @@
+
+   What is Scrum?
+
+        In simple words Scrum is a lightweight yet incredibly powerful set of values, principles and practices.
+
+   Definition:
+
+        Scrum: An[agile] framework within which people can address complex adaptive problems, while productively and creatively delivering products of the highest possible value.
+
+   Difference Between Scrum & Agile:
+
+        The difference between agile and Scrum is that agile refers to a set of principles and values shared by several methodologies, processes and practices; Scrum is one of several 
+	agile frameworks--and is the most popular.
+
+   Benefits of Using Scrum:
+   
+        As Scrum teams are continuously creating only the highest priority chunks of functionality, the business is assured maximum return on investment.
+
+
+   Scrum helps businesses:
+
+             * Innovate faster
+             * Move from idea to delivery more quickly
+             * Drive higher customer satisfaction
+             * Increase employee morale
+
+
+   Scrum defines three roles
+
+   1.PRODUCT OWNER : 
+    
+               The Product Owner defines the what--as in what the product will look like and what features it should contain. 
```

Comment:
Indent should be properly done.

Example: 
your text:

The Product Owner is expected to incorporate stakeholder feedback to create the highest value product increments each and every sprint. 
	       Product Owners maintain the product backlog and ensures that everyone knows the priorities.

I want : 
The Product Owner is expected to incorporate stakeholder feedback to create the highest value product increments each and every sprint. 
Product Owners maintain the product backlog and ensure that everyone knows the priorities.